src/Products/urban/indexes.py

```
@@ -275,7 +276,13 @@ def genericlicence_decisiondate(licence):
                 return decision_date
         if linked_pm_items:
             if "date" in linked_pm_items[0]["extra_include_meeting"].keys():
-                return linked_pm_items[0]["extra_include_meeting"]["date"]
+                decision_date = linked_pm_items[0]["extra_include_meeting"]["date"]
+                if isinstance(decision_date, unicode):
```

Ruff (0.15.12)

[error] Undefined name `unicode`

(F821)